modules/local/sra_fastq_ftp.nf

```diff
@@ -31,17 +32,17 @@ process SRA_FASTQ_FTP {
     script:    
     if (meta.single_end) {
         """
-        curl -L ${fastq[0]} -o ${meta.id}.fastq.gz
+        timeout $task.time bash -c 'until curl $options.args -L ${fastq[0]} -o ${meta.id}.fastq.gz; do sleep 1; done';

```

Comment:
```suggestion
        bash -c 'until curl $options.args -L ${fastq[0]} -o ${meta.id}.fastq.gz; do sleep 1; done';

```

Reply:
Ok. Last question! Just thought about this but why do we need the `timeout` if Nextflow will kill the job after `$task.time` anyway?

Reply:
Yup, just tested locally and we don't explicitly need `timeout $task.time` in the command either. I think it's better for NF to kill the process when it wants without having to manually specify this unless you can think of another advantage?

Reply:
why this? caus the timeout is covered by the process now?

Reply:
Yes, exactly. NF will kill the process when `$task.time` is reached anyway. That is the same behaviour you were trying to introduce there right?

Reply:
I agree with this